src/main/scala/is/hail/variant/VariantSampleMatrix.scala

```diff
@@ -269,6 +269,42 @@ class VariantSampleMatrix[RPK, RK, T >: Null](val hc: HailContext, val metadata:
 
   lazy val sampleAnnotationsBc = sparkContext.broadcast(sampleAnnotations)
 
+  def rdd2: RDD[RegionValue] = {
+    val localRowSignature = rowSignature
+    val localGSignature = genotypeSignature
+    val localNSamples = nSamples
+    rdd.mapPartitions { it =>
+      val region = MemoryBuffer()
+      val rvb = new RegionValueBuilder(region)
+      val rv = RegionValue(region)
+
+      val a = new Array[Any](localNSamples)
```

Comment:
??

Reply:
What's the question?  `a` is the array that's storing the row that's going to be kicked out.

Reply:
I don't see any references to `a`. Just booted up IntelliJ to verify, definitely no usages. It doesn't look like you kick out rows any more, you're kicking out `RegionValue`.

Reply:
Oops!  I was looking at the wrong line.